internal/provider/datasource_directory_role_collection_test.go

```diff
@@ -44,7 +44,7 @@ func TestDataSourceDirectoryRoleCollection(t *testing.T) {
 			Steps: []resource.TestStep{
 				{
 					Config:      hclProviderFor(user) + hclDatasourceDirectoryRoleCollection("uut", "5357bda0-8651-4eab-a69d-12d282bc3247", "Directory Viewer"),
-					ExpectError: regexp.MustCompile(`Received response with unexpected status \[Status: 400; Correlation ID:\s+[a-f0-9\-]+\]`),
+					ExpectError: regexp.MustCompile(`Received response with unexpected status \[Status: 403; Correlation ID:\s+[a-f0-9\-]+\]`),
```

Comment:
I don't think, 403 should be called unexpected. In the tested case the directory is not security enabled. There is also a server message that should be printed so that the user can react to the actual problem.

Reply:
Done